client/src/domain/ReservationView/style.scss

```diff
@@ -30,3 +30,25 @@
     }
   }
 }
+@media screen and (max-width: 800px) {
+  .reservation {
+    & h4 {
+      font-size: 20px;
+      margin-bottom: 20px;
+    }
+    & h5 {
+      font-size: 20px;
+      margin-bottom: 16px;
+    }
+    &__form {
+      & input {
+        font-size: 0.7rem;
+        max-width: 200px;
+      }
+      & h5 {
+        margin-bottom: 20px;
+        font-size: 16px;
+      }
```

Comment:
don't mix rem and px values (if possible). If mb is 20px it equals to 1.25rem etc